pkg/.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
these files are artifacts from the files common to the ongoing PRs, and leftover from a different directory structure; will remove